pokemongo_bot/__init__.py

```diff
@@ -423,6 +424,28 @@ def _read_config_location(self):
         logger.log(u'[x] Address found: {}'.format(self.config.location))
 
     def _get_pos_by_name(self, location_name):
+        if location_name.count(',') == 1:
+            try:
+                logger.log("[x] Fetching altitude from google")
+                parts = location_name.split(',')
+
+                pos_lat = float(parts[0])
+                pos_lng = float(parts[1])
+
+                # we need to ask google for the altitude
+                gmaps = googlemaps.Client(key=self.config.gmapkey)
+                response = gmaps.elevation((pos_lat, pos_lng))
+
+                if len(response) and "elevation" in response[0]:
+                    return pos_lat, pos_lng, response[0]["elevation"]
+                else:
+                    raise ValueError
+            except ApiError:
+                logger.log("[x] Could not fetch altitude from google. Trying geolocator.")
+            except ValueError:
+                logger.log("[x] Location was not Lng/Lat.")
```

Comment:
bit of a nitpick, but mind switching to "[x] Location was not **Lat/Lng**"? (bold used as diff) 

Same with comment below